src/main/java/org/stocker/routes/UpdateRecordsRoute.java

```diff
@@ -41,6 +45,16 @@ public UpdateRecordsRoute(StockDataClient stockDataClient, StockDBClient stockDB
     @Override
     public void handle(YokeRequest yokeRequest, Handler<Object> handler) {
         YokeResponse response = yokeRequest.response();
+        StockMarket market = null;
+        if (yokeRequest.getParameter("market").equalsIgnoreCase(StockMarket.INDIA.toString())) {
+            market = StockMarket.INDIA;
+        } else if (yokeRequest.getParameter("market").equalsIgnoreCase(StockMarket.US.toString())){
+            market = StockMarket.US;
+        } else {
+            response.setStatusCode(400);
+            response.end(ErrorObjectAdapter.generateErrorObj(ServiceException.ILLEGAL_REQUEST));
```

Comment:
Did not know this, thought end would end the flow. I'll be careful in future. Also, I might just remove this and default to one of the markets is not sent